pcs/resource.py

```diff
@@ -838,6 +879,7 @@ def resource_unmove_unban(lib, argv, modifiers):
     resource_id = argv.pop(0)
     node = argv.pop(0) if argv else None
 
+    _check_is_not_stonith(lib, [resource_id])
```

Comment:
Can this be moved to a library command? Then we could simply replace deprecation warning with an error in order to actually forbid moving stonith resources.